app/controllers/events/view/tickets/order-form.js

```diff
@@ -0,0 +1,30 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+  /**
+   * Save the forms.
+   */
+  async saveForms(data) {
+    for (const customForm of data.customForms ? data.customForms.toArray() : []) {
+      await customForm.save();
+    }
+    return data;
+  },
+
+  actions: {
+    save(data) {
+      this.set('isLoading', true);
+      this.saveForms(data)
+        .then(() => {
+          this.get('notify').success(this.get('l10n').t('Your Attendee form has been saved'));
+        })
+        .catch(e => {
+          console.error(e);
+          this.get('notify').error(this.get('l10n').t('Oops something went wrong. Please try again'));
```

Comment:
@srv-twry I was thinking of adding the error message on notify banner itself. IMO a normal user will not check his console. What do you think?

Reply:
Yeah, cool. Will add in the next PR for this feature if you don't mind. :)